js/query.js

```diff
@@ -0,0 +1,69 @@
+(function() {
+        document.addEventListener("DOMContentLoaded", function(event) {
+            console.log("DOM fully loaded.");
+            document.getElementById("searchButton").addEventListener('click', newQuery);
+        });
+
+        function newQuery() {
+            var term = document.getElementById("query").value;
+            var query_url = "https://mygene.info/v3/query?q=" + term;
+            var species_options = document.getElementById("species");
+            if (species_options.selectedIndex !== -1) {
```

Comment:
1. Add a space between variable declarations and business logic.
2. Use camelCasing in JavaScript.